### PR TITLE
Configuration for usage with Github Enterprise Server

### PR DESCRIPTION
**Pull request by ephraimduncan**

### CODE REVIEW

File: src/providers/github.ts [hunk 1]
```diff
@@ -14,8 +11,18 @@ export class GitHub implements OAuth2Provider {
 		clientSecret: string,
 		options?: {
 			redirectURI?: string;
+			/** Configuration for usage with [GitHub Enterprise Server](https://docs.github.com/en/enterprise-server/get-started). */
+			enterprise?: {
```
Comment (pilcrowonpaper) [inline src/providers/github.ts]: Can you rename it to just `enterpriseDomain` to be consistent with the other providers? Or was there a reason to make it an object?

Comment (ephraimduncan) [reply src/providers/github.ts]: not exactly, i will rename it

**Comment by pilcrowonpaper** [comment]
Looks good, can you update the docs too?

**Review by pilcrowonpaper — APPROVED** [review]

**Review by pilcrowonpaper — APPROVED** [review]

**Comment by pilcrowonpaper** [comment]
Can you create a new changeset?
https://github.com/pilcrowOnPaper/arctic/blob/main/CONTRIBUTING.md

**Comment by ephraimduncan** [comment]
@pilcrowOnPaper Added the changeset

**Review by pilcrowonpaper — APPROVED** [review]

**Comment by pilcrowonpaper** [comment]
Thanks!

**Pull request merged**